src/stockwatch/use_cases/degiro.py

```diff
@@ -2,14 +2,22 @@
 import threading
 from dataclasses import dataclass
 from datetime import date, timedelta
+from http import HTTPStatus
 
 import requests
+from requests.api import head
 
 from stockwatch.use_cases.configuring import Config
 
 from . import stockdir
 
 
+def __get_headers() -> dict[str, str]:
```

Comment:
why the leading double underscore? I would prefer a single one

Reply:
double underscore is super private, as in python automatically adds some stuff in front so other people really cannot import it, see also this stackoverflow: https://stackoverflow.com/a/1301369

I do agree it might not really be necessary for this method.